<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="Django default" uuid="9dc15dd8-4cab-4e48-98ab-0760a4c4af33">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/DjangoGirls/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\Users\otavi\Desktop\DjangoGirls\db.sqlite3</jdbc-url>
      <driver-properties>
        <property name="enable_load_extension" value="true" />
      </driver-properties>
    </data-source>
    <data-source source="LOCAL" name="db" uuid="2fe8babc-1e97-4c69-a5d6-0ca17af3d133">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>SQLite</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\Users\otavi\OneDrive\Documentos\Projetos\PYTHON\DjangoGirls\db.sqlite3</jdbc-url>
      <driver-properties>
        <property name="enable_load_extension" value="true" />
      </driver-properties>
    </data-source>
  </component>
</project>